src/main/java/gg/projecteden/nexus/features/chat/alerts/AlertsCommand.java

```diff
@@ -52,16 +52,26 @@ void list() {
 			JsonBuilder builder = new JsonBuilder();
 
 			if (highlight.isPartialMatching()) {
-				builder.next("&a✔")
-						.command("/alerts partialmatch false " + highlight.getHighlight())
-						.hover("&cClick to turn off partial matching");
+				builder.group().next("&d⦿ ")
```

Comment:
I think the UI here does not really fit with how we normally do things. With multiple small buttons, it may be best to put it in a book, or if we want to keep it in chat my best suggestion is this: https://i.imgur.com/tY0X1m1.png
Either way, we do not normally use &d in chat anywhere, so I think it should be one of our normal colors (a, c, e, 3)